projects/firefox/target.c

```diff
@@ -48,9 +62,16 @@ int main(int argc, char* argv[]) {
 
   // Temporary (or permanent?) work-arounds for fuzzing interface bugs.
   char* options = getenv("ASAN_OPTIONS");
+  const char max_alloc_size_opt[] = ":max_allocation_size_mb=2049";
```

Comment:
are you sure that would work? ASan might be parsing this on the startup only.

there is a way to specify ASan options via `.options` file: https://github.com/google/oss-fuzz/blob/2f4da4e8da7fff37ae3140397d61c38fc17b6b10/projects/wasmtime/default.options#L1

Reply:
That is much better. Is that in the docs somewhere?

Reply:
Doesn't seem so. I think we removed `.options` files from documentation when deprecated `-max_len`. As for the sanitizer options, this is really advanced usage and the vast majority of our users don't need it, so I'm not even sure if it's worth documenting or not.

Reply:
OK, that makes me feel better :)
I looked in the docs earlier today hoping to avoid this and I was worried I'd missed it. I agree it is pretty advanced, so yeah that's fair.

I'm testing locally and I will push an update soon, thanks.

Reply:
Sorry for the inconvenience. I'm slightly debating inside myself on whether it's still better to document it, at least in CF docs (since `.options` file in a feature of ClusterFuzz), and then we can have a link from OSS-Fuzz docs to there.

@jonathanmetzman @oliverchang WDYT? Should CF docs have a page describing `.option` file syntax and what can be done with it?

Reply:
Also would prefer not to document it, since it's rare that people really needed and it can be easily abused.